### DISCUSSION TITLE
Feature Request: Threshold for Sequential Failures

### DISCUSSION THREAD

**Discussion by meyersbs** [upptime/upptime#715] [Ideas]
Since the `HTTP Code: 0` response is a [known issue](https://github.com/upptime/upptime/issues/171) that doesn't appear to be getting attention, it would be great if we had a `__sequential_failures_down: 2` config option, that would only mark a site as down (and open an issue) if there were multiple error codes in a row.

**Comment by AnandChowdhary** [comment]
We already do this and only open an issue if we get an error three times in a row: https://github.com/upptime/uptime-monitor/blob/master/src/update.ts#L261
